optional-container-engine/test/background.test.js

```diff
@@ -13,415 +13,386 @@
-        `logging.error() should have been called`
-      );
-      assert.deepEqual(
-        mocks.logging.error.firstCall.args,
-        [`Error occurred while getting pubsub topic`, testErrorMsg],
-        `logging.error() should have been called with the right arguments`
-      );
-    });
-    it(`should log error if cannot publish message`, () => {
-      // Setup
-      const testBookId = 1;
-      const testErrorMsg = `test error`;
-      mocks.topic.publish = sinon.stub().callsArgWith(1, testErrorMsg);
+test.serial(`should log error if cannot get topic`, (t) => {
+  // Setup
+  const testBookId = 1;
+  const testErrorMsg = `test error`;
+  mocks.pubsub.createTopic = sinon.stub().callsArgWith(1, testErrorMsg);
 
-      // Run target functionality
-      background.queueBook(testBookId);
+  // Run target functionality
+  background.queueBook(testBookId);
 
-      // Assertions
-      assert.ok(
-        mocks.pubsub.createTopic,
-        `pubsub.createTopic() should have been called once`
-      );
-      assert.equal(
-        mocks.pubsub.createTopic.firstCall.args[0],
-        `book-process-queue`,
-        `pubsub.createTopic() should have been called with the right arguments`
-      );
-      assert.equal(
-        mocks.pubsub.topic.callCount,
-        0,
-        `pubsub.topic() should NOT have been called`
-      );
-      assert.ok(
-        mocks.topic.publish,
-        `topic.publish() should have been called once`
-      );
-      assert.deepEqual(
-        mocks.topic.publish.firstCall.args[0],
-        {
-          data: {
-            action: `processBook`,
-            bookId: testBookId
-          }
-        },
-        `topic.publish() should have been called with the right arguments`
-      );
-      assert.equal(
-        mocks.logging.info.callCount,
-        0,
-        `logging.info() should NOT have been called`
-      );
-      assert.ok(
-        mocks.logging.error.calledOnce,
-        `logging.error() should have been called`
-      );
-      assert.deepEqual(
-        mocks.logging.error.firstCall.args,
-        [`Error occurred while queuing background task`, testErrorMsg],
-        `logging.error() should have been called with the right arguments`
-      );
-    });
-  });
+  // Assertions
+  t.truthy(
+    mocks.pubsub.createTopic,
+    `pubsub.createTopic() should have been called once`
+  );
+  t.is(
+    mocks.pubsub.createTopic.firstCall.args[0],
+    `book-process-queue`,
+    `pubsub.createTopic() should have been called with the right arguments`
+  );
+  t.is(
+    mocks.pubsub.topic.callCount,
+    0,
+    `pubsub.topic() should NOT have been called`
+  );
+  t.is(
+    mocks.topic.publish.callCount,
+    0,
+    `topic.publish() should NOT have been called`
+  );
+  t.is(
+    mocks.logging.info.callCount,
+    0,
+    `logging.info() should NOT have been called`
+  );
+  t.truthy(
+    mocks.logging.error.calledOnce,
+    `logging.error() should have been called`
+  );
+  t
```

Comment:
Typo? (If so, make sure there aren't any further instances of this throughout your PR.)